.github/workflows/rules.yml

```diff
@@ -41,12 +45,16 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install pytest-cov
-        pip install .[${{inputs.pip-extras}}]
-        if [[ "${{ inputs.environment }}" == "qibo" ]]; then
+        if [ -z ${{ inputs.pip-extras }} ]; then
+          pip install .
+        else
+          pip install .[${{ inputs.pip-extras }}]
+        fi
```

Comment:
Ok, it might be clearer like this, but `pip install .[ ]` was actually doing the same as `pip install .`.

Reply:
Not really, check [this out](https://github.com/qiboteam/qibojit/actions/runs/4087883690/jobs/7048925819#step:4:56).

Reply:
I think wrapping ` .[ ]` with `".[ ]"` works now.

Reply:
> I think wrapping ` .[ ]` with `".[ ]"` works now.

Ok, this is exactly what I tried locally, because `zsh` interprets square brackets (specifically I was using single quotes to avoid this). I just thought that `bash` did not create this kind of issues, maybe it was naive.